.env

```diff
@@ -4,6 +4,6 @@ DATABASE_USER=docker
 DATABASE_PASSWORD=docker
 DATABASE_DB=dataline
 DATABASE_URL=jdbc:postgresql://db:5432/dataline
-CONFIG_ROOT=data/config
+CONFIG_ROOT=/data
```

Comment:
allows us to reuse this string constant in docker-compose and as the root in java. means that we need to add the `/config` in `DefaultConfigPersistence`, which is a little weird, but still better than adding extra env variable that needs to match up with one of the existing ones. (e.g. `CONFIG_ROOT=data/config` and `DATA_MOUNT=data`)